src/cljc/athens/common_events/resolver.cljc

```diff
@@ -276,10 +276,10 @@
 (defmethod resolve-event-to-tx :datascript/unindent-multi
   [db {:event/keys [args]}]
   (println "resolver :datascript/unindent-multi args" args)
-  (let [{:keys [uids
-                first-uid]}              args
+  (let [{:keys [uids]}              args
+        [uid-if-embed-block _]      (common-db/uid-and-embed-id (first uids))
```

Comment:
embed-id's should not be used in resolver, they are like entity id's - local to the client.
You probably need this embed-id to be used in event followup.

Reply:
Actually, looking at `events.cljs` I see that you've already sanitised those uids, so you shouldn't need to process embed-uids here.

Reply:
yeah 🤦🏽